local-setup/.env.dev

```diff
@@ -26,6 +26,8 @@ NODE_ENV=local
 # The following key/token are MANDATORY to give when running worker.
 # Otherwise request-vc might suffering from data provider error.
 TWITTER_AUTH_TOKEN_V2=
+TWITTER_CLIENT_ID=twitter_client_id
+TWITTER_CLIENT_SECRET=twitter_client_secret
```

Comment:
Should these two parameters be put somewhere else? I think they are not keys we need to protect?